libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -469,10 +484,14 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
 
   #_descriptorProperty = 'name';
 
+  #_enableShowMore = false;
+
   #_highlightText: string[] | undefined;
 
   #_inputDirective: SkyAutocompleteInputDirective | undefined;
 
+  #_isDropdownVisible = new BehaviorSubject<boolean>(false);
```

Comment:
We reserve the `#_` syntax for get/set properties. I think we'd be safe to do the following:
```
protected isDropdownVisible: Observable<boolean>;

#isDropdownVisible = new BehaviorSubject<boolean>(false);
```

Reply:
Updated.